packages/auto-label/test/auto-label.ts

```diff
@@ -841,6 +841,10 @@ describe('auto-label', () => {
         {title: 'com.google.spanner.helper: ignored', want: 'api: spanner'},
         {title: 'fix(snippets.spanner.helper): ignored', want: 'api: spanner'},
         {title: 'snippets.video: ignored', want: 'api: videointelligence'},
+        {
+          title: 'data-science-onramp.video-intelligence.ignored: ignored',
+          want: 'api: videointelligence',
```

Comment:
I think this should be `ai-platform`?

Reply:
I used Video Intelligence (see `title`) because AI Platform isn't included in the test resources.

See https://github.com/googleapis/repo-automation-bots/blob/master/packages/auto-label/test/fixtures/events/downloadedfile.json.

Reply:
Ah it's for a test, didn't see that. My bad 😅